benchto-driver/src/main/java/com/teradata/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -47,6 +48,8 @@
 public class BenchmarkServiceExecutionListener
```

Comment:
maybe add base class `AsnynchroniousBenchmarkExecutionListener` so that async listeners don't have to implement large closures (which are not nicest to read)?